### DISCUSSION TITLE
Help version

### DISCUSSION THREAD

**Discussion by TchaleeB** [Desuuuu/klipper#128] [Q&A]
Hello, I'm a little lost on the versions. Is the latest updated version on the reloaded repo or on the repo display ? 
The date of the reloaded repo is the latest but the latest firmware for the display must be installed with the version of klipper that is on the display repo as I understand it. 
Thanks in advance.

**Comment by Desuuuu** [comment]
The proper display firmware is linked in the README for the `dgus-reloaded` branch.

All the information about the `dgus-display` branch (which is more recent), is available in [this post](https://github.com/Desuuuu/klipper/discussions/74).

**Reply by TchaleeB** [reply]
Ok so that's what I understood 😅 Thanks for the answer.
